keras_cv/layers/preprocessing/random_shear.py

```
@@ -59,6 +65,7 @@ def __init__(
         fill_mode="reflect",
         fill_value=0.0,
         seed=None,
+        bounding_box_format="xyxy",
```

Bounding box format should default to “None” to prevent silent errors